scripts/uploadSchools/package.json

```diff
@@ -11,6 +11,7 @@
   "dependencies": {
     "dotenv": "^8.2.0",
     "firebase": "^7.9.1",
+    "lodash": "^4.17.15",
```

Comment:
What was the reasoning for adding this? We dont use the general `lodash` package in that file.

Reply:
When I ran **node index.js** command it gave me an error which said I needed to install it.

Reply:
Lets remove this change and just run `npm install` in this directory to install the lodash packages we need

Reply:
do I revert that change using git or just uninstall?

Reply:
it should be good now

Reply:
Looks good :+1: